### PR TITLE
start postgres instead of calling the replicant role

### PR DESCRIPTION
**Pull request by dtwilliamson**

### CODE REVIEW

File: swap_database.yml [hunk 1]
```diff
@@ -96,10 +96,12 @@
         state: touch
     # The replicant service should still be stopped at this point.
```
Comment (reedstrm) [inline swap_database.yml]: Hmm, for non-single-server environments, doesn't the postgresql service need to be restarted on replicant?

Comment (dtwilliamson) [inline swap_database.yml]: I forgot to add a `hosts:` clause. Thanks.

**Comment by dtwilliamson** [comment]
Actually, I should probably just remove the touch step, since it's not needed at all.

**Comment by pumazi** [comment]
So, replication setup for the swapped in database is now a manual operation?

**Comment by dtwilliamson** [comment]
No, swa_database.yml includes a pg_basebackup step and everything else necessary. If there's a better approach to making all this work, I'd like to know. I'm in the process of testing the changes, btw.

**Comment by pumazi** [comment]
I thought the replicant would end up unconfigured, which is why the role is called. Doesn't pg_basebackup blow away the existing data directory?

**Comment by dtwilliamson** [comment]
The first step of the role requires that postgres is running in order to respond to the query. Besides, the replicant role has already been run when the replicant server was provisioned. At that time, though, the primary database is empty since it's newly provisioned. copy_database.yml copies the db from prod to staging, for example, as dbname_new on the primary and does nothing with the replicant. swap_database.yml, swaps dbname to _old and _new to dbname on the primary then does pg_basebackup from the primary to the secondary. So I wouldn't say the replicant is unconfigured since it was previously provisioned and then becomes a duplicate of the primary.

**Pull request merged**